tools/apidiff/delta/delta.go

```diff
@@ -46,6 +47,20 @@ func GetExports(lhs, rhs exports.Content) exports.Content {
 		}
 	}
 
+	structFields := GetStructFields(lhs, rhs)
+	if len(structFields) > 0 {
```

Comment:
>structFields [](start = 8, length = 12)

hmmm so if there is a new field we would say its a brekaing change ? would be better to do it just if the field is not a pointer ? If so is there  a way to check if its a pointer or not ?

Reply:
No, this just includes new struct fields when diff'ing the sides (same goes for the interface method code I added below).